homeassistant/components/risco/entity.py

```diff
@@ -29,17 +15,10 @@ def _refresh_from_coordinator(self):
     async def async_added_to_hass(self):
         """When entity is added to hass."""
         self.async_on_remove(
-            self._coordinator.async_add_listener(self._refresh_from_coordinator)
+            self.coordinator.async_add_listener(self._refresh_from_coordinator)
```

Comment:
won't this result in two calls to `async_write_ha_state()` (one directly from `CoordinatorEntity` and the other from `RiscoEntity._refresh_from_coordinator()`?

Reply:
not if super isnt called in risco method

Reply:
Good call!